src/org/zaproxy/zap/extension/viewstate/ViewStateModel.java

```diff
@@ -0,0 +1,328 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
```

Comment:
I would say that it should be updated, this is also a new class with new content.